### ISSUE TITLE
babel 7 support

### ISSUE THREAD

**Issue by NeoPhi** [chpio/babel-plugin-transform-async-to-bluebird#4]
Are there any plans to release a beta version of this plugin that works with babel 7?

**Comment by lostfictions** [comment]
For people landing here, it looks like [`@babel/transform-async-to-generator`](https://babeljs.io/docs/en/next/babel-plugin-transform-async-to-generator.html) might let you do the same thing as this plugin?

https://babeljs.io/docs/en/next/v7-migration#babel-plugin-async-to-generator

```diff
{
  "plugins": [
-    ["@babel/transform-async-to-module-method"]
+    ["@babel/transform-async-to-generator", {
+      "module": "bluebird",
+      "method": "coroutine"
+    }]
  ]
}
```

**Comment by chpio** [comment]
> might let you do the same thing as this plugin?

No, it's not the same thing, my plugin checks if there's `await`-ing going on in the code, and switches to the more lightweight `bluebird.method` function. The problem here is the increased code-complexity for the checking and thus the effort of an update... i should maybe just mark it as deprecated in favor of the mentioned babel plugin.

edit: i should deprecated it in favor of the standard `Promise` impl, not bluebird.
